content/statefulset/services.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Create Services"
+date: 2018-08-07T08:30:11-07:00
+weight: 15
+---
+#### Introduction
+[Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/) defines a logical set of Pods and a policy by which to access them. Service can be exposed in different ways by specifying a [type](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/) in the serviceSpec. By specifying **"None"** for the clusterIP, you can create [headless service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services) required by StatefulSets.
+#### Create Services
+Copy/Paste the following commands into your Cloud9 Terminal.
+```
+cd ~/environment/templates
+wget https://eksworkshop.com/statefulset/services.files/mysql-services.yml
+```
+Check the configuration of mysql-services.yml by following command.
+```
+cat ~/environment/templates/mysql-services.yml
+# Headless service for stable DNS entries of StatefulSet members.
```

Comment:
If you want to provide example output, break it into a separate code box so it doesn't get copy/pasted with the cat command. There are examples elsewhere of wording like "The output should look something like this"